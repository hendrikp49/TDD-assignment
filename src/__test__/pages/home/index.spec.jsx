import Home from "@/components/pages/home";
import { render } from "@testing-library/react";

describe("Home Component", () => {
  it("render home component", () => {
    const { container } = render(<Home />);
    expect(container).toMatchSnapshot();
  });
});
