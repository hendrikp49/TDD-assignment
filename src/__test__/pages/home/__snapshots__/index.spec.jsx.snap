// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home Component render home component 1`] = `
<div>
  <div>
    test
  </div>
  <button />
</div>
`;
